7/debian9/7.6/Dockerfile

```diff
@@ -49,7 +49,20 @@ RUN set -x \
  && cd /usr/local/src \
  && git clone --depth=1 https://github.com/SonarSource/sonarqube.git \
  && rm -rf /opt/sonarqube/bin/*
+COPY license-checksum /
 
+RUN  set -x \
+   && mkdir /opt/tmp \
+   && wget -q -O /opt/tmp/LICENSE "https://raw.githubusercontent.com/dmeiners88/sonarqube-prometheus-exporter/develop/LICENSE" \
```

Comment:
How do you know that this LICENSE matches this JAR?

Reply:
Because JAR file is in release https://github.com/dmeiners88/sonarqube-prometheus-exporter/releases and LICENSE places in https://github.com/dmeiners88/sonarqube-prometheus-exporter/blob/develop/LICENSE

Reply:
Please set matching License to JAR (commit).